src/blake2s.ts

```diff
@@ -6,9 +6,7 @@ import { rotr, toBytes, wrapConstructorWithOpts, u32 } from './utils.js';
 // first 32 bits of the fractional parts of the square roots of the first 8 primes 2..19)
 // same as SHA-256
 // prettier-ignore
-export const IV = new Uint32Array([
-  0x6a09e667, 0xbb67ae85, 0x3c6ef372, 0xa54ff53a, 0x510e527f, 0x9b05688c, 0x1f83d9ab, 0x5be0cd19
-]);
+export const IV = /* @__PURE__ */new Uint32Array([0x6a09e667, 0xbb67ae85, 0x3c6ef372, 0xa54ff53a, 0x510e527f, 0x9b05688c, 0x1f83d9ab, 0x5be0cd19]);
```

Comment:
does it make sense to keep this pure? it's exported.

Reply:
This annotation make sense. I can import only `compress` function from this file and  `Uint32Array([...])` would be removed otherwise it will in bundle with args